Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
+import CardBanner from './CardBanner';
+
+const CardContainer = () => {
+    return (
+        <div>
+            <CardBanner />
```

Comment:
You would probably want to include `CardContent` here too. It doesn't look like you added that component though.